.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -1,34 +1,34 @@
 name: Bug report
 description: Use this template for reporting bugs. Please search existing issues first.
 labels: bug, needs-triage
+type: Bug
 body:
-  - type: markdown
+  - type: textarea
     attributes:
-      value: '## Detailed issue description'
```

Comment:
I removed section headings as they take a bit too much space without adding a lot of value